ajax/libs/pagecss/package.json

```diff
@@ -23,7 +20,11 @@
 			"files": [
 				"**/*"
 			]
-		}],
+		}],	
+		"keywords": [
+		"pagecss",
+		"css",
```

Comment:
`,` after "css" should be removed, maybe you should consider to lint it before commit new changes (jsonlint.com)